docs/best_practice/crate_ami.txt

```diff
@@ -0,0 +1,90 @@
+.. highlight:: yaml
+.. _crate_ami:
+
+================================
+Launch a custom Crate AMI on AWS
+================================
+
+Crate AWS Images are public AMIs (see `AMI`_) and can be launched from every
```

Comment:
Crate provides custom AMIs that can be launched ... using the AWS Management Console or AWS CLI